src/Api/Auth/Controllers/WebAuthnController.cs

```diff
@@ -75,6 +82,17 @@ public async Task Post([FromBody] WebAuthnCredentialRequestModel model)
         }
     }
 
+    private async Task ValidateRequireSsoPolicyDisabledOrNotApplicable(Guid userId)
+    {
+        var requireSsoLogin = await _policyService.AnyPoliciesApplicableToUserAsync(userId, PolicyType.RequireSso);
+
+        if (requireSsoLogin)
+        {
+            // User is not an admin or owner of this org and the org requires SSO login so do not allow passkey creation.
```

Comment:
suggestion(non-blocking): this comment is not relevant anymore, right?

Reply:
No, it is still relevant. The logic is just applied via the `AnyPoliciesApplicableToUserAsync` method.

Reply:
Ok but since we're not actually checking for admin or owner in this class, someone could change the implementation of `AnyPoliciesApplicableToUserAsync` and not have any idea that this comment needed to be updated? 

My point was mostly that we don't need to worry about those things here, since the service takes care of them for us

Reply:
Ah, fair. Ok, I'll remove the comment. Thank you for the discussion